code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -51,7 +51,7 @@
 		to_chat(user, "<span class='warning'>There is no power left in the shard.</span>")
 		return
 
-	if(!ishuman(M) || istype(M, /mob/living/carbon/human/dummy)) //If target is not a human or a dummy
+	if(!ishuman(M) || istype(M, /mob/living/carbon/human/dummy) || istype(M, /mob/living/carbon/human/manifested)) //If target is not a human or a dummy

```

Comment:
Neither of these should be here; manifested ghosts are meant to be cultists, and, as such, can't be put into a soul stone, as per below.